### ISSUE TITLE
Issue getting JSON data to work on package

### ISSUE THREAD

**Issue by nikcio** [umbraco/Umbraco.Marketplace.Issues#8]
I'm trying to use the new JSON data for the Umbraco marketplace but it doesn't seem to show on the marketplace website.

Package: https://marketplace.umbraco.com/package/nikcio.uheadless

JSON file: https://github.com/nikcio/Nikcio.UHeadless/blob/v3-contrib/umbraco-marketplace-nikcio.uheadless.json

I first thought the default branch name was the issue. I prefer using `v3/contrib` but changed it to `v3-contrib` to try and solve the issue but It doesn't seem to make a difference.

**Note: I've change my default branch name back to `v3/contrib`. Let me know if I should change it to `v3-contrib`**

**Comment by AndyButland** [comment]
Hi @nikcio - thanks for letting us know about the problem.  I've had a quick look and can't see anything wrong with the setup, but clearly something is up if it's not picking up.  I'm travelling today but tomorrow I'll investigate and try to get this sorted.

**Comment by AndyButland** [comment]
I think we've got this working now.  Seemed to be an issue we'd missed when adding several related packages, I've run the update on your package and looks to be coming over OK.

One small thing, not really related, but I noticed we aren't properly getting your author image.  I think as the URL you have triggers a redirect to [here](https://avatars.githubusercontent.com/u/24605285?v=4).  Should be OK if you update to use that URL directly.
